ci/infra/testrunner/utils/utils.py

```diff
@@ -155,35 +157,34 @@ def ssh_sock_fn(self):
     @timeout(60)
     @step
     def setup_ssh(self):
-
         self.runshellcommand("chmod 400 " + self.conf.ssh_key_option)
-        print("Starting ssh-agent ")
         # use a dedicated agent to minimize stateful components
         sock_fn = self.ssh_sock_fn()
         try:
             self.runshellcommand("pkill -f 'ssh-agent -a {}'".format(sock_fn))
-            print("Killed previous instance of ssh-agent")
+            logger.warning("Killed previous instance of ssh-agent")
         except:
             pass
         self.runshellcommand("ssh-agent -a {}".format(sock_fn))
-        print("adding id_shared ssh key")
         self.runshellcommand("ssh-add " + self.conf.ssh_key_option, env={"SSH_AUTH_SOCK": sock_fn})
 
     @timeout(30)
     @step
     def info(self):
         """Node info"""
-        print("Env vars: {}".format(sorted(os.environ)))
-
-        self.runshellcommand('ip a')
-        self.runshellcommand('ip r')
-        self.runshellcommand('cat /etc/resolv.conf')
+        info_lines  = "Env vars: {}\n".format(sorted(os.environ))
+        info_lines += self.runshellcommand('ip a')
+        info_lines += self.runshellcommand('ip r')
+        info_lines += self.runshellcommand('cat /etc/resolv.conf')
```

Comment:
Running cat as a shell command is a little ugly.  Why not
```suggestion
        with open("/etc/resolv.conf", "r") as resolv
            info_lines += resolv.read()
```

Reply:
I guess that'd likely also need a try block, and should probably be a separate PR which cleans up some other system commands which have native python equivalents (like `rm`).  Maybe nevermind. :)

Reply:
Opening an issue for this.